src/QxManager/EmptyRow.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types=1);
+
+namespace MLL\Utils\QxManager;
+
+class EmptyRow
+{
+    public function toString(): string
+    {
+        return implode(QxManagerSampleSheet::DELIMITER, [
+            'No',
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
```

Comment:
I would prefer a solution that uses https://www.php.net/manual/en/function.array-fill.php to make the count obvious, I guess it is tied to a constant?